docs/users/docker_installation.md

```diff
@@ -1,6 +1,6 @@
 ## macOS Installation: Docker Desktop for Mac
 
-Docker Desktop for Mac can be installed via Homebrew (`brew cask install docker`) or can be downloaded from [download.docker.com](https://download.docker.com/mac/stable/Docker.dmg).
+Docker Desktop for Mac can be installed via Homebrew (`brew install --cask docker`) or can be downloaded from [download.docker.com](https://download.docker.com/mac/stable/Docker.dmg).
```

Comment:
Actually, I don't think this is correct. The new usage to get docker desktop is `brew install homebrew/cask/docker` - Doing what you have here just gets the docker client.